chia/_tests/wallet/test_new_wallet_protocol.py

```diff
@@ -278,7 +278,7 @@ async def test_request_coin_state(one_node: OneNode, self_hostname: str) -> None
         coinbase=False,
         timestamp=uint64(1),
     )
-    await add_coin_records_to_db(simulator.full_node.coin_store.db_wrapper, [*coin_records, ignored_coin])
+    await add_coin_records_to_db(simulator.full_node.coin_store, [*coin_records, ignored_coin])
```

Comment:
This repeated line caused type errors. One mild refactor moves the hack into a common area. Not ideal, but works for this very special `CoinStore` case.